build/nuget-release.yml

```diff
@@ -106,10 +106,33 @@ stages:
             parameters:
               dotnetSdkVersion: '$(DotNet.Sdk.Version)'
               projectName: '$(Project).Tests.Integration'
+              category: 'Docker'
+          - task: PowerShell@2
+            displayName: 'Get Docker container logs'
+            inputs:
+              targetType: 'inline'
+              script: 'docker logs $(Images.ServiceBus.Queue)'
+            condition: failed()
+
+  - stage: SelfContainingIntegrationTests
+    displayName: Self-Containing Integration Tests
+    dependsOn: Build
+    condition: succeeded()
+    jobs:
+      - job: SelfContqiningIntegrationTests
+        displayName: 'Run self-containing integration tests'
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - template: test/run-integration-tests.yml@templates
```

Comment:
If this is all that's need to run the tests, does that mean the Docker stage does the same? Which means also test the self-contained tests?

Reply:
No, I've updated the category of the tests to 'Docker' so it's only running those tests.
The 'self-containing' or 'in code self-hosted' tests are under a different category.